.github/workflows/package-build.yml

```diff
@@ -86,17 +140,16 @@ jobs:
         echo "$BUILD_SDK_PATH"
         echo "###### xcode-select Info ######"
         xcode-select -print-path
-        echo "###### LLVM Info ######"
```

Comment:
Why not keeping this message?

Reply:
Don't remember exactly but probably did that because the llvm-config call was removed. Since with newer llvm versions and it's CMake files it's less relevant. It's probably not even in path at this point.

Reply:
Restored the header since there are still some commands printing info about LLVM.